tests/unit/ng/event/test_set_event_backlog.py

```diff
@@ -0,0 +1,86 @@
+# pylint: disable=missing-docstring
+
+import pytest
+
+from logprep.ng.abc.event import Event
+from logprep.ng.event.event_state import EventStateType
+from logprep.ng.event.set_event_backlog import SetEventBacklog
+
+
+class DummyEvent(Event):
+    __slots__ = Event.__slots__
```

Comment:
We want to use the backlog for concrete implementations of the Event. Perhaps it is useful to use the concrete implementations for testing the backlogs?

consider use LogEvent, SreEvent and PseudonymEvent in this testcases. We have to ensure, that this breaks if someone changes these concrete implementations in a unpredictable way.